### ISSUE TITLE
[Question] How to Configure Backup RPCs

### ISSUE THREAD

**Issue by Planxnx** [erpc/erpc#111]
I’m looking for example config for set up backup RPCs in eRPC. 
Here’s the scenario:
- I have 1 self-hosted node and 1 pay-as-you-go node (e.g. QuickNode).
- The goal is to use the self-hosted node as the primary and only fall back to the pay-as-you-go node(backup node) if the primary goes down.

Could you share an example or provide instructions on how to configure this setup?

**Comment by aramalipoor** [comment]
@Planxnx there are 2 ways to control fallback behavior for such purpose:

1. To use the `upstream.scoreMultipliers` to make your self-hosted node always the preferred on request-level (first try that node and if request fails immediately retry on others) https://docs.erpc.cloud/config/projects/upstreams#customizing-scores--priorities

In this method all upstreams are always available but starts from your own nodes.

---

2. Is to use `network.selectionPolicy` as described here: https://docs.erpc.cloud/config/projects/selection-policies

In summary if you assign `group: fallback` to any of your upstreams a "default" selection policy is activated where it exclude these fallback nodes and ONLY add them if other upstreams (you self-hosted node) are down.

In this method runs on some interval  (5 minutes) and decides which upstreams to have active or disable completely. I recommend to use this one with caution and proper testing.

If you need any further help or question feel free to post here or join the telegram group: https://t.me/erpc_cloud
